src/components/SignupForm.jsx

```diff
@@ -41,45 +50,81 @@ const SubmitButton = styled.button`
   border-radius: 4px;
   cursor: pointer;
 `;
+const StyledLink = styled(Link)`
+  color: #007bff;
+  text-decoration: none;
+  margin-left: 10px;
+`;
```

Comment:
Move the styles to a separate style.js file and export the components you want to use so you can access them by importing them. Try to make use of reusability